templates/vanilla/packages/contracts/deploys/31337/latest.json

```diff
@@ -1,4 +1,4 @@
 {
   "worldAddress": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
-  "blockNumber": 9
+  "blockNumber": 10
```

Comment:
should we git ignore this file?

Reply:
whoops, it was ignored but forgot to delete